plugins/woocommerce/includes/admin/settings/class-wc-settings-page.php

```diff
@@ -31,6 +31,42 @@ abstract class WC_Settings_Page {
 		 */
 		public $icon = 'settings';
 
+		/**
+		 * Settings field types which are known.
+		 *
+		 * @var string
```

Comment:
Perhaps array type here
```suggestion
		 * @var string[]
```